src/template.rs

```diff
@@ -90,10 +90,21 @@ pub struct Crate {
 
 #[derive(Debug, Serialize)]
 pub enum Source {
-    CratesIo { sha256: String },
```

Comment:
Prefer not to bleed whitespace changes in. Probably disable your auto format. I'll add a rustfmt configuration if we don't have one

:shipit:  Let's merge this, with or without whitespace. Ping me for re-approval if necessary

Reply:
@psionic-k :ping_pong: